docs/00-getting-started/00_getting-started.jmd

```diff
@@ -0,0 +1,78 @@
+---
+title: Getting Started
+permalink: /:collection/:name/
```

Comment:
Would these fields of `permalink` and `redirect_from` still work with `jmd`?

Reply:
I copied this structure from all the files in `TuringTutorials/tutorials`, but I am assuming this is read by the build process. Perhaps @devmotion could also advise here?